scripts/install.js

```diff
@@ -92,37 +92,47 @@ function download(url, dest, cb) {
  */
 
 function checkAndDownloadBinary() {
-  if (sass.hasBinary(sass.getBinaryPath())) {
+  if (process.env.SKIP_SASS_BINARY_DOWNLOAD_FOR_CI) {
+    console.log('Skipping downloading binaries on CI builds');
```

Comment:
This was already here, it has just been moved. The npmlog issue is a separate concern.